### PR TITLE
lz4: Add version 1.9.3

### LINKED ISSUE #3179: [Request] lz4
### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [x] a non-GUI tool
- [x] a reasonably well-known and widely used developer tool
- [x] the latest stable version of the program
- [x] the full version i.e. not a trial version
- [x] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** lz4

**Description:** Command-line compressor/decompressor for LZ4. LZ4 is lossless compression algorithm, providing compression speed > 500 MB/s per core, scalable with multi-cores CPU. It features an extremely fast decoder, with speed in multiple GB/s per core, typically reaching RAM speed limits on multi-core systems.

**Homepage:** https://github.com/lz4/lz4

**Download link(s):** https://github.com/lz4/lz4/releases

**Some indication of popularity/repute:** 6.7k GitHub stars

### PR DESCRIPTION
**Pull request by kvadevack**
Closes #3179

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/lz4.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.9.3",
+    "description": "Fast LZ compression algorithm library - tool",
```
Comment (rashil2000) [inline bucket/lz4.json:3]:
```suggestion
    "description": "Fast LZ compression algorithm library and tool",
```

File: bucket/lz4.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "homepage": "https://github.com/lz4/lz4",
```
Comment (rashil2000) [inline bucket/lz4.json:4]:
```suggestion
    "homepage": "https://www.lz4.org",
```

File: bucket/lz4.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        "64bit": {
+            "url": "https://github.com/lz4/lz4/releases/download/v1.9.3/lz4_win64_v1_9_3.zip",
+            "hash": "519cb3bfc4975cce8747b1ecba5053137b6ce2a77b653e4d2c63fae057985449"
+        },
+        "32bit": {
+            "url": "https://github.com/lz4/lz4/releases/download/v1.9.3/lz4_win32_v1_9_3.zip",
+            "hash": "4eb78a7374b0ccadd46d6a148b42257e56890b5a1cceaf4db40d543281ad037c"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\*\" -Recurse -Force -Exclude lz4.exe, README.md, NEWS",
```
Comment (rashil2000) [inline bucket/lz4.json:16]: Why delete rest of the folders? Are they not helpful (even if not directly required)?

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by kvadevack** [comment]
Incorporated your feedback @rashil2000

**Pull request merged**